copy/copy.go

```diff
@@ -361,6 +365,44 @@ func supportsMultipleImages(dest types.ImageDestination) bool {
 	return false
 }
 
+// compareImageDestinationManifestEqual compares the `src` and `dest` image's manifests, returning the 'true' if they compare equal.
+// The destination manifest is always fetched.
```

Comment:
This is not the case on error paths. “This usually fetches the destination manifest”? Or maybe just remove this comment.